themes/finna2/templates/RecordTab/holdingsils.phtml

```diff
@@ -203,7 +216,12 @@
     )
   ?>
 <?php endif; ?>
-
+<?php
+// If holdings tab visible for archive request form, other items not shown.
+if ($allowArchiveRequest) {
+  return;
```

Comment:
Onko tosiaan näin? Ettei mitään saatavuustietoja näytetä? Aina?

Reply:
Tämä edelleen epäselvä. Se, sallitaanko varaus, ei liity siihen, onko saatavuustietoja näytettävänä.